src/features/search/components/SearchResults/SearchResults.tsx

```diff
@@ -93,9 +93,9 @@ export const SearchResults: React.FC = () => {
   const { headerTransition: scrollButtonTransition, onScroll } = useOpacityTransition()
 
   const { params } = useRoute<UseRouteType<'Search'>>()
-  // TODO(refacto recherche): after removing location feature flag this can should be deleted
+  // TODO(PC-25239): after removing location feature flag this can should be deleted
```

Comment:
(Optionnel)
Peut être refaire la phrase en disant "this should be deleted" parce que je suis pas convaincue par le "can should"